ci-scripts/integration-test-disable-gui.sh

```diff
@@ -77,7 +77,8 @@ echo "starting $COIN node in background with http listener on $HOST"
             -launch-browser=false \
             -data-dir="$DATA_DIR" \
             -wallet-dir="$WALLET_DIR" \
-            -enable-api-set=ALL \
+            -enable-all-api-sets=true \
```

Comment:
My first impression is that the fact that the option name is `--enable-all-api-sets` but not all API sets are enabled , might lead to confusion . `--enable-supported-api-sets` ?

Reply:
The API sets that it does not enable are special endpoints that wouldn't be enabled normally, `INSECURE_WALLET_SEED` is only intended for the desktop client, and for `DEPRECATED_WALLET_SPEND` they need to stop using it.

If it enabled all endpoints it would do the correct behavior for no user.  In no case, except for the integration tests, would anyone want to enable every endpoint including `INSECURE_WALLET_SEED` and `DEPRECATED_WALLET_SPEND`.

`-enable-supported-api-sets` doesn't provide any more clarity, because the meaning of `supported` is ambiguous.  I think any name cannot be expressive enough, so it is described in the doc string.